README.markdown

```diff
@@ -396,19 +396,29 @@ SSL handshake if the `wss://` scheme is used.
 
 * `client_cert`
 
-    Specifies a client certificate chain cdata object that will be used while TLS handshaking with remote server. 
-    These objects can be created using 
-    [ngx.ssl.parse_pem_cert](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/ssl.md#parse_pem_cert) 
-    function provided by lua-resty-core. 
+    Specifies a client certificate chain cdata object that will be used while TLS handshaking with remote server.
+    These objects can be created using
+    [ngx.ssl.parse_pem_cert](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/ssl.md#parse_pem_cert)
+    function provided by lua-resty-core.
     Note that specifying the `client_cert` option requires corresponding `client_priv_key` be provided too. See below.
 
 * `client_priv_key`
 
-    Specifies a private key corresponds to the `client_cert` option above. 
-    These objects can be created using 
-    [ngx.ssl.parse_pem_priv_key](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/ssl.md#parse_pem_priv_key) 
+    Specifies a private key corresponds to the `client_cert` option above.
+    These objects can be created using
+    [ngx.ssl.parse_pem_priv_key](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/ssl.md#parse_pem_priv_key)
     function provided by lua-resty-core.
 
+* `host`
+
+    Specifies the value of the `Host` header sent in the handshake request. If not provided, the `Host` header will be derived from the hostname/address and port in the connection URI.
+
+* `server_name`
+
+    Specifies the server name (SNI) to use when performing the TLS handshake with the server. If not provided, the `host` value or the `<host/addr>:<port>` from the connection URI will be used.
+
+
+
```

Comment:
2 extra lines? If we are to maintain this as an OpenResty patch, there should not be any issue that will cause the patch to be rejected upstream.